### PR TITLE
UI adjustments

### PR DESCRIPTION
**Pull request by Zeitsperre**
### Pull Request Checklist:
- [x] `pre-commit` hooks are installed/active in my local clone (`$ pre-commit install`)
- [ ] This PR addresses an already opened issue (for bug fixes / features)
    - This PR fixes #xyz
- [ ] (If applicable) Documentation has been added / updated (for bug fixes / features)
- [ ] If a merge request has been made in parallel to this PR in xscen-notebooks, it is merged and the submodules have been updated.
- [ ] HISTORY.rst has been updated (with summary of main changes)
  - [ ] Link to issue (:issue:`number`) and pull request (:pull:`number`) has been added

### What kind of change does this PR introduce?

* Adds a bit of versatility to the allowed inputs for Data/ProjectCatalogs

### Does this PR introduce a breaking change?

No.

### CODE REVIEW

File: xscen/config.py [hunk 1]
```diff
@@ -108,6 +111,16 @@ def recursive_update(d, other):
     return d
 
 
+def args_as_str(*args: Tuple[Any, ...]) -> Tuple[str, ...]:
```
Comment (RondeauG) [inline xscen/config.py:114]: If the goal is to allow passing a `Path` to `DataCatalog`, wouldn't returning a tuple break it?

Comment (Zeitsperre) [reply xscen/config.py:114]: I'm fairly certain that the data type for a function using `*args` is `Tuple[Any, ...]`, hence why accessing `arg` from `*args` is typically done within a loop/indexing. I might be wrong, though.

Comment (RondeauG) [reply xscen/config.py:117]: Right, that's indeed the case.

**Review by RondeauG — APPROVED** [review]
I'll let you update the History and such, but otherwise it looks good! I also merged the latest 'main' with the new docs.

**Pull request merged**